css-layout-classes/styles.css

```diff
@@ -0,0 +1,29 @@
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  font-family: Arial, sans-serif;
+}
+
+.row {
+  display: flex;
+}
+
+.reversed {
+  flex-direction: row-reverse;
+}
+
```

Comment:
Just as a minor thing to think about:

While it is good that you thought to make a separate utility class for your `flex-direction`, you may just want to specify the class such as `row-reversed` since if you were to ever need a `flex-direction: column-reverse`, you would likely name it `col-reversed`. Just something to think about since the `reversed` class is specified to `rows`